cmake/License.rtf

```diff
@@ -7,14 +7,14 @@
 \pard\pardeftab720\partightenfactor0
 
 \f0\fs26 \cf2 \expnd0\expndtw0\kerning0
-SPDX-License-Identifier: Apache-2.0\
+Copyright 2010-2020 The Khronos Group Inc.\
+SPDX-License-Identifier: Apache-2.0
```

Comment:
All text lines end in \\. I think the lack here is why the macos package build is failing.

Reply:
Not sure what to do about that. I directly edited the .rtf file because editing it in Libreoffice produced about 10x the volume of markup in the output, and I don't know RTF markup. Maybe you could edit it to insert the copyright/license using whatever editing tool you normally use, and attach it to the thread here?

Reply:
OK, just adding the trailing \ seems to work as far as libreoffice is concerned - NM.